File: app/src/main/java/com/stevenfrew/beatprompter/bluetooth/PauseOnScrollStartMessage.java
package com.stevenfrew.beatprompter.bluetooth;

public class PauseOnScrollStartMessage extends SignalOnlyBluetoothMessage
{
    static final byte PAUSE_ON_SCROLL_START_MESSAGE_ID=3;

    public PauseOnScrollStartMessage()
    {
        super(PAUSE_ON_SCROLL_START_MESSAGE_ID);
    }

    PauseOnScrollStartMessage(byte[] bytes) throws NotEnoughBluetoothDataException
    {
        super(bytes);
    }
}

File: app/src/main/kotlin/com/stevenfrew/beatprompter/bluetooth/PauseOnScrollStartMessage.kt
package com.stevenfrew.beatprompter.bluetooth

class PauseOnScrollStartMessage : SignalOnlyBluetoothMessage(PAUSE_ON_SCROLL_START_MESSAGE_ID) {
    companion object {
        internal const val PAUSE_ON_SCROLL_START_MESSAGE_ID: Byte = 3
    }
}
